conf/vmware.yaml.template

```
@@ -1,7 +1,16 @@
 VMWARE:
   # VMware to be added as a compute resource
   # VCENTER: vmware vcenter URL
-  VCENTER:
+  VCENTER7:
+    HOSTNAME:
+    HYPERVISOR:
+    # mac_address: Mac address of the automation-robottelo-vmware-vm
```

vm_name we're not defining here in template
```suggestion
    # mac_address: Mac address of the vm_name
```